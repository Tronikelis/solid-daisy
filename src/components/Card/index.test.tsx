import { render } from "@solidjs/testing-library";
import { describe, expect, it } from "vitest";

import { Card } from ".";

describe("<Card />", () => {
    it("renders correctly", () => {
        const { container } = render(() => <Card>Foo</Card>);
        expect(container).toMatchSnapshot();
    });
});
