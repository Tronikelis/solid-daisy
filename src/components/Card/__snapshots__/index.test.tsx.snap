// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Card /> > renders correctly 1`] = `
<div>
  <div
    class="rounded-2xl bg-base-100 p-4 shadow-xl border border-gray-200"
  >
    Foo
  </div>
</div>
`;
